sdk/python/feast/infra/online_stores/contrib/postgres.py

```diff
@@ -172,7 +182,9 @@ def online_read(
             # when we iterate through the keys since they are in the correct order
             values_dict = defaultdict(list)
             for row in rows if rows is not None else []:
-                values_dict[row[0].tobytes()].append(row[1:])
+                values_dict[
+                    row[0] if isinstance(row[0], bytes) else row[0].tobytes()
+                ].append(row[1:])
```

Comment:
Only call `tobytes()` when `row[0]` is not already of bytes type. Otherwise, this will result in Errors.